### PR TITLE
add notes functionality.

### PR DESCRIPTION
**Pull request by drets**
@rasendubi, frontend for the notes done
(expects `notes` field to be an array of strings from backend, updating/getting data − similar to `tags`)

### CODE REVIEW

**Comment by drets** [comment]
(frontend shows only the first note)

**Comment by rasendubi** [comment]
Is it safe to merge if the backend does not support notes yet?

**Comment by drets** [comment]
Yes, it will show the notes icon which won't save the notes till the backend ready, no errors should occur.

File: src/components/ArticlesTable/ArticlesTable.js [hunk 1]
```diff
@@ -41,15 +48,17 @@ export class ArticlesTable extends Component {
     super(props);
 
     this.state = {
-      deleteArticleId: null,
+      ...notesInitialState,
+      deleteArticleId: 0,
```
Comment (rasendubi) [inline src/components/ArticlesTable/ArticlesTable.js:52]: `0` is a valid article id. If I understand correctly, I won't be able to delete it.

**Review by rasendubi — COMMENTED** [review]
Possible UI improvements.

Show notes icon in next cases only:
1. there are saved notes
2. an article is hovered by mouse

Also, show number of notes.

**Comment by drets** [comment]
> Also, show number of notes.

It should go after we extend the functionality to show the all notes.

File: src/components/ArticlesTable/ArticlesTable.js (hunk 1 above)
Comment (drets) [reply src/components/ArticlesTable/ArticlesTable.js:52]: :+1: 

`s/0/-1/`

**Comment by drets** [comment]
(the notes icon changes brightness when there are notes)

**Pull request merged**